responders/src/UI/embc-responder/src/app/feature-components/wizard/step-create-ess-file/step-create-ess-file.component.html

```diff
@@ -22,7 +22,7 @@
         'tab-margin custom-tab-link': tabItem.name !== 'review'
       }"
     >
-      <span class="span-height" (click)="isAllowed(tabItem.name, $event)"
+      <span class="span-height" (click)="isAllowed(tabItem.route, $event)"
```

Comment:
'route' is the incorrect property to be used here. The correct property that should be used is 'name'

Reply:
Then why is the property called tabRoute in the isAllowed method?

Reply:
Routes can change, name property is more reliable should be used at both tab and step level. These changes are WIP, are you using this function somewhere?

Reply:
IsAllowed is used when checking if the Review tab can be accessed. If we're consistently passing the name into it, then the first parameter in that function should be changed to tabName or tabRouteName.

Reply:
Please leave it as is, this is WIP, I am working on it.

Reply:
Okay, didn't realize you were.